app/Models/Calendar.php

```diff
@@ -704,6 +704,18 @@ public function upcomingReminders(int $needle = 10): Collection
                                     ->whereNull('recurring_until')
                                     // Events that end in the future are fine, they could be reoccuring on this month
                                     ->orWhere('recurring_until', '>=', $this->currentYear());
+                            })
```

Comment:
Seems to be missing from the `pastReminders()` block on the calendar dashboard widget.